theano/gpuarray/elemwise.py

```diff
@@ -202,7 +184,7 @@ def c_init_code_struct(self, node, name, sub):
                            typecode=o.type.typecode)
```

Comment:
This area does need some changes, but p is still required to keep track of which args we are at.